util.ts

```diff
@@ -57,3 +83,30 @@ export function uint8arrayToArrayBuffer(input: Uint8Array): ArrayBuffer {
 export function stringToUtf8(text: string): ArrayBuffer {
 	return uint8arrayToArrayBuffer(new TextEncoder().encode(text));
 }
+export function getUserDNPFormat(){
+	// @ts-expect-error : Internal Method
+	const dailyNotePluginInstance = app.internalPlugins.getPluginById("daily-notes").instance;
```

Comment:
app refers to global - we should use a proper reference.

Reply:
don't I need to use global app to get an internal plugin? What's the alternative?

Reply:
You can get an explicit reference to app via this.app on your FormatImporter.